examples/pwscf/kccsd.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from pyscf.pbc import cc
+from pyscf.pbc.pwscf.pw_helper import gtomf2pwmf
+from pyscf.pbc.pwscf.kccsd_rhf import PWKRCCSD
+
+"""
+Simple CCSD calculation
+"""
+
+a0 = 1.78339987
+atom = "C 0 0 0; C %.10f %.10f %.10f" % (a0*0.5, a0*0.5, a0*0.5)
+a = np.asarray([
+        [0., a0, a0],
+        [a0, 0., a0],
+        [a0, a0, 0.]])
+
+from pyscf.pbc import gto, scf, pwscf
+cell = gto.Cell(atom=atom, a=a, basis="gth-szv", pseudo="gth-pade",
+                ke_cutoff=50)
+cell.build()
+cell.verbose = 5
+
+kpts = cell.make_kpts([2,1,1])
+
+mf = scf.KRHF(cell, kpts)
+mf.kernel()
+
+mcc = cc.kccsd_rhf.RCCSD(mf)
+mcc.kernel()
+
+pwmf = gtomf2pwmf(mf)
+pwmcc = PWKRCCSD(pwmf).kernel()
```

Comment:
pwkrccsd can be started from a regular `pwscf.PWKRHF(cell, kpts)` calculation, right? I think this "normal" usage can be demonstrated . And the conversion between the GTO-KHF can then be demonstrated, with some explanation comments.

Reply:
Yes, I can add these examples and comments.